disnake/user.py

```diff
@@ -417,7 +452,22 @@ class User(BaseUser, disnake.abc.Messageable):
     id: :class:`int`
         The user's unique ID.
     discriminator: :class:`str`
-        The user's discriminator. This is given when the username has conflicts.
+        The user's discriminator.
+
+        .. note::
+            This is being phased out by Discord; the username system is moving away from ``username#discriminator``
+            to users having a globally unique username.
+            The value of a single zero (``"0"``) indicates that the user has been migrated to the new system.
+            See the `help article <https://dis.gd/app-usernames>`__ for details.
+
+    global_name: Optional[:class:`str`]
+        The user's global display name, if set.
+        This takes precedence over :attr:`.name` when shown.
+
+        For bots, this is the application name.
```

Comment:
just double checked and this is currently not true 🦐

Reply:
yeah :/ It was announced back then, which is why I originally included it in this PR. It'll be correct eventually :p
![image](https://github.com/DisnakeDev/disnake/assets/8530778/65d613a9-5770-41ea-9461-e99a10a16a3e)
<sup>[source](https://canary.discord.com/channels/613425648685547541/697138785317814292/1103385812969721967)</sup>